tests/test_keras_integration.py

```diff
@@ -440,62 +478,66 @@ def test_override_tensorboard(self):
+                repo_id=repo_id,
                 log_dir=f"{tmpdirname}/tb_log_dir",
                 api_endpoint=ENDPOINT_STAGING,
-                use_auth_token=self._token,
-                git_user="ci",
-                git_email="ci@dummy.com",
+                token=self._token,
             )
+
             os.makedirs(f"{tmpdirname}/tb_log_dir2")
             with open(f"{tmpdirname}/tb_log_dir2/override.txt", "w") as fp:
                 fp.write("Keras FTW")
             push_to_hub_keras(
                 model,
-                repo_path_or_name=f"{WORKING_REPO_DIR}/{REPO_NAME}",
+                repo_id=repo_id,
                 log_dir=f"{tmpdirname}/tb_log_dir2",
                 api_endpoint=ENDPOINT_STAGING,
-                use_auth_token=self._token,
-                git_user="ci",
-                git_email="ci@dummy.com",
+                token=self._token,
             )
 
-            model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(
-                f"{USER}/{REPO_NAME}",
-            )
+            model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(repo_id)
+            # TODO: this test fails. Both tensorboard.txt and override.txt are in the
+            #       repo at the end.
             self.assertTrue(
                 "logs/override.txt" in [f.rfilename for f in model_info.siblings]
             )
             self.assertFalse(
                 "logs/tensorboard.txt" in [f.rfilename for f in model_info.siblings]
             )
 
-            self._api.delete_repo(repo_id=f"{REPO_NAME}", token=self._token)
+            self._api.delete_repo(repo_id=repo_id, token=self._token)
 
     @retry_endpoint
-    def test_push_to_hub_model_kwargs(self):
-        REPO_NAME = repo_name("PUSH_TO_HUB")
+    def test_push_to_hub_keras_via_http_with_model_kwargs(self):
+        REPO_NAME = repo_name("PUSH_TO_HUB_KERAS_via_http_with_model_kwargs")
+        repo_id = f"{USER}/{REPO_NAME}"
+
         model = self.model_init()
         model = self.model_fit(model)
         push_to_hub_keras(
             model,
-            repo_path_or_name=f"{WORKING_REPO_DIR}/{REPO_NAME}",
+            repo_id=repo_id,
             api_endpoint=ENDPOINT_STAGING,
-            use_auth_token=self._token,
-            git_user="ci",
-            git_email="ci@dummy.com",
-            config={"num": 7, "act": "gelu_fast"},
+            token=self._token,
             include_optimizer=True,
             save_traces=False,
         )
 
-        model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(
-            f"{USER}/{REPO_NAME}",
-        )
-        self.assertEqual(model_info.modelId, f"{USER}/{REPO_NAME}")
+        model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(repo_id)
+        self.assertEqual(model_info.modelId, repo_id)
 
-        from_pretrained_keras(f"{WORKING_REPO_DIR}/{REPO_NAME}")
-        self.assertRaises(ValueError, msg="Exception encountered when calling layer*")
```

Comment:
I am not used to unittest (and `self.assertRaises`) but I think this test never worked as intended.
I tried it manually and the `from_pretrained_keras` never fails for me. From [this discussion](https://github.com/huggingface/huggingface_hub/pull/616/files#r796563431), I understand that the goal was to make it fail with a custom layer.

Reply:
Note: I changed it to 
```python
with self.assertRaises(ValueError, msg="Exception encountered when calling layer*" ):
    from_pretrained_keras(tmpdirname)
```
in the tests so now the test fails (no ValueError thrown)

Reply:
Ah indeed. Thanks for fixing it!